### ISSUE TITLE
Can not install apps through BHBB: errors/crashing

### ISSUE THREAD

**Issue by ralphieb2t** [Ibrahim778/BetterHomebrewBrowser#31]
When I choose to download an application and it downloads right away, my vita crashes, sometimes while it’s still spinning, sometimes mid-download. If it is added to the que, it fully downloads, but when I am prompted to either ‘install’ or ‘save file’ and I try to install, I always get a “could not install[E-afffffe]” and the action is not completed. 

Never had this issue with the old HBB; Any ideas what the issue might be? Thanks.

**Comment by Ibrahim778** [comment]
I'm going to be treating this as 2 separate problems, those being, crashes while queuing and / or downloading and cannot install
Could you please list the following:
- Your FW version
- Your tai config.txt file
- Do you have iTLS-enso?
- Storage configuration & space remaining
- The app you are trying to download 

If you could provide a video showcasing the crashes and install errors that would be helpful
The error you have given is an insufficient storage space error, quite strange it does not tell you, it should have shown another popup explaining there is low space and how much more you need.

Another thing you can try is disabling the Download Enabler plugin if you have it as it is known to sometimes conflict, so disable the plugin in your config, reboot, and see if that fixes the crashes.

**Comment by ralphieb2t** [comment]
> I'm going to be treating this as 2 separate problems, those being, crashes while queuing and / or downloading and cannot install Could you please list the following:
> 
> * Your FW version
> * Your tai config.txt file
> * Do you have iTLS-enso?
> * Storage configuration & space remaining
> * The app you are trying to download
> 
> If you could provide a video showcasing the crashes and install errors that would be helpful The error you have given is an insufficient storage space error, quite strange it does not tell you, it should have shown another popup explaining there is low space and how much more you need.
> 
> Another thing you can try is disabling the Download Enabler plugin if you have it as it is known to sometimes conflict, so disable the plugin in your config, reboot, and see if that fixes the crashes.

Ok, so I just saw this. Let me at least answer the questions I know off the top of my head right away. 
> Pch-1000, FW 3.65. 
> I have the full compatibility pack of ITLS installed. 
> Using SD2vita— which I have set as Uxo, Uro is the internal. And I’m not sure of the exact space off hand, but should be plenty; as a rule, I do not ever go under 20-30GB’s just to prevent that type of thing. So that’s very strange if the error is correct; otherwise no there was no other popup. 
> I believe the App that I tried to test download was “Custom Live Area Vita.”
-which btw I had no problem installing manually & using. 
I'll get you the Config & some more specifics when I can get on my vita, is there anything else you’d like me to look at while I’m at it? I know I for sure have DownloadEnabler on there though, so perhaps that’s what’s causing issues.

**Comment by ralphieb2t** [comment]
Ok, so I just saw this. Let me at least answer the questions I know off the top of my head right away. 
> * Pch-1000, FW 3.65. 
> * I have the full compatibility pack of ITLS installed. 
> * Using SD2vita— which I have set as Uxo, Uro is the internal. And I’m not sure of the exact space off hand, but should be plenty; as a rule, I do not ever go under 20-30GB’s just to prevent that type of thing. So that’s very strange if the error is correct; otherwise no there was no other popup. 
> * I believe the App that I tried to test download was “Custom Live Area Vita.”
- which btw I had no problem installing manually & using. 
I'll get you the Config & some more specifics when I can get on my vita, is there anything else you’d like me to look at while I’m at it? I know I for sure have DownloadEnabler on there though, so perhaps that’s what’s causing issues.

**Comment by Ibrahim778** [comment]
So I just tested downloading the app myself, and I was able to reproduce the issue, I'm going to look into it now.

Just to clarify, BHBB still *saves* the file for you doesn't it? Because if an install fails that is what should happen.
Also, have you tried to download any other app and see if it others work fine? That'd narrow it down.

**Comment by Ibrahim778** [comment]
Try using the new release version 1.2
Fixed it for me.

**Comment by ralphieb2t** [comment]
> 

No kidding! So what, it was the one app I decided to try? What are the chances. Yes, I’d get the error msg, it wouldn’t install, but the VPK would still be properly downloaded in its folder; however, this was only when I downloaded through VHBB’s queue. 

(Issue2) If I downloaded normally, watching the progress bar I mean, without the use of the background service, my Vita would crash before it was even finished. Did you by any chance experience that as well? At least we should know wether that’s a separate issue now though. Lmk and then I guess I can always test something else, I figured I’d better stop for the time being since I didn’t know what was going on before though.

**Comment by Ibrahim778** [comment]
Yes, it was that one app specifically.

As for your second issue I'm really quite confused, if the download starts then it's not a DownloadEnabler conflict (still disable it just to be sure).
Also, the app will _always_ download using the background service. Not sure why it's crashing in the middle of the download though I've literally never heard of that before, it sounds like a problem in Sony's code.
Still waiting on your config.txt

**Comment by ralphieb2t** [comment]
Hello again, hmm Interesting. Yeah, If I just let it download with the download bar window on screen— it always crashes, either midway through its progress, or it will get to the end & immediately crash. I guess I’ll have to try downloading a different app with the DownloadEnabler disabled. Also here is a pic of the Config.txt in my tai folder: 
![DFA4CBDB-97F8-4F5A-84BF-D41502C1C231](https://github.com/Ibrahim778/BetterHomebrewBrowser/assets/73906926/e91b09d7-2863-4e80-83d0-cf5f26f7ca50)
![96FCA8CC-E5CB-43AF-A7FC-3B16370B63B5](https://github.com/Ibrahim778/BetterHomebrewBrowser/assets/73906926/5d2bf4ad-091d-4674-8bdf-c7865cca0480)

**Comment by Ibrahim778** [comment]
Yeah try disabling DownloadEnabler and see if that fixes it

**Comment by Ibrahim778** [comment]
Closing due to inactivity.
